src/app/main/data/table/table.component.scss

```diff
@@ -16,6 +16,7 @@ example-component {
 }
 
 table-basic {
+  height: calc(100% - 67px);
```

Comment:
Why this height? the table basic screen keeps showing up  unnecessary scroll  
![image](https://user-images.githubusercontent.com/32088627/225247830-fb693465-3f11-4264-98a8-05b3b73f2169.png)

Reply:
https://github.com/OntimizeWeb/ontimize-web-ngx-playground/issues/185